uaclient/cli.py

```diff
@@ -901,14 +901,19 @@ def action_config_set(args, *, cfg, **kwargs):
     ):
         # setup_apt_proxy is destructive for unprovided values. Source complete
         # current config values from uaclient.conf before applying set_value.
-        if set_key in cfg.deprecated_global_scoped_proxy_options:
-            print(messages.WARNING_APT_PROXY_SETUP)
-            set_key = "global_" + set_key
-        protocol_type = set_key.split("_")[2]
+
+        protocol_type = "https" if "https" in set_key else "http"
         if protocol_type == "http":
             validate_url = util.PROXY_VALIDATION_APT_HTTP_URL
+            s = ""
         else:
             validate_url = util.PROXY_VALIDATION_APT_HTTPS_URL
+            s = "s"
+
+        if set_key in cfg.deprecated_global_scoped_proxy_options:
+            print(messages.WARNING_APT_PROXY_SETUP.format(s=s))
```

Comment:
I think we can drop the `s` variable and just pass the `protocol_ type` directly to the string right ?

We would not need to turn it into:
`Warning: apt_{protocol_type}_proxy has been renamed to global_apt_{protocol_type}_proxy`

Reply:
yep thanks! thats much simpler, I'll make that change

Reply:
updated and pushed!